src/Effects.js

```diff
@@ -1445,6 +1445,40 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.music_wave = {
+      wave: {},
+      init: function () {
+        this.wave = {
+          inc: 360/15,
+          heightFactor: 1,
+          heightDivider: 200,
+          yLoc: 300,
+          xTotal: 400,
+        };
+      },
+
+      draw: function (context) {
+        const centroid = context.centroid;
+        let scale = p5.map(centroid, 5000, 8000, 0, 250);
+        let angle = 0;
+        p5.background('black');
+        const wave = this.wave;
+        for (let i = 1; i <= wave.xTotal; i+=5) {
```

Comment:
Nit-pick (i.e. not worth blocking this from shipping, but worth thinking about) - It looks like the way this loop works is that it is 'looping over' the pixel width of the p5 draw space, drawing a new line every five pixels. It might be clearer to 'loop over' the number of lines instead (something like `for (let i = 0; i < 80; i++) {`). Either way should make the same drawing, so it just depends on what makes the most sense to you.

Reply:
Since the value of i is used when drawing the line, I used a variable for the lineWidth. I think this makes the code more flexible to changes in the size of the drawing area.